physics/terminal_velocity.py

```diff
@@ -0,0 +1,60 @@
+from scipy.constants import g
+
+"""
+Title : Computing the terminal velocity of an object falling
+        through a fluid.
+
+Terminal velocity is defined as the highest velocity attained by an
+object falling through a fluid. It is observed when the sum of drag force
+and buoyancy is equal to the downward gravity force acting on the
+object. The acceleration of the object is zero as the net force acting on
+the object is zero.
+
+Vt = ((2 * m * g)/(ρ * A * Cd))^0.5
+
+where :
+Vt = Terminal velocity (in m/s)
+m = Mass of the falling object (in Kg)
+g = Acceleration due to gravity (value taken : imported from scipy)
+ρ = Density of the fluid through which the object is falling (in Kg/m^3)
+A = Projected area of the object (in m^2)
+Cd = Drag coefficient (dimensionless)
+
+Reference : https://byjus.com/physics/derivation-of-terminal-velocity/
+"""
```

Comment:
```suggestion
"""
Title : Computing the terminal velocity of an object falling
        through a fluid.

Terminal velocity is defined as the highest velocity attained by an
object falling through a fluid. It is observed when the sum of drag force
and buoyancy is equal to the downward gravity force acting on the
object. The acceleration of the object is zero as the net force acting on
the object is zero.

Vt = ((2 * m * g)/(ρ * A * Cd))^0.5

where :
Vt = Terminal velocity (in m/s)
m = Mass of the falling object (in Kg)
g = Acceleration due to gravity (value taken : imported from scipy)
ρ = Density of the fluid through which the object is falling (in Kg/m^3)
A = Projected area of the object (in m^2)
Cd = Drag coefficient (dimensionless)

Reference : https://byjus.com/physics/derivation-of-terminal-velocity/
"""

from scipy.constants import g
```

Generally the docstring should be at the very top of the file, before the imports

Reply:
okay sure , I'll keep this in mind

Reply:
I had another pull request in which I had used the value of g as 9.8 I'll change the value there as well